api/tests/core/offers/test_repository.py

```diff
@@ -1437,3 +1437,110 @@ def test_exclude_offers_from_inactive_venue_provider(self):
         assert offer_from_stock_api in selected_offers
         assert offer_from_inactive_venue_provider not in selected_offers
         assert offer_from_deleted_venue_provider not in selected_offers
+
+
+@pytest.mark.usefixtures("db_session")
+class GetStocksListFiltersTest:
+    def test_basic(self):
+        stock = factories.EventStockFactory()
+
+        # When
+        stocks = repository.get_filtered_stocks(
+            offer_id=stock.offer.id,
+            stocks_limit_per_page=20,
+        )
+
+        # Then
+        assert len(stocks) == 1
+
+    def test_filtered_stock_by_price_category(self):
+        # Given
+        stock = factories.EventStockFactory()
+        factories.EventStockFactory()
+
+        # When
+        stocks = repository.get_filtered_stocks(
+            offer_id=stock.offer.id,
+            stocks_limit_per_page=20,
+            price_category_id=stock.priceCategoryId,
+        )
+
+        # Then
+        assert len(stocks) == 1
+
+    @freeze_time("2020-10-15 00:00:00")
+    def test_filtered_stock_by_date(self):
+        # Given
+        now = datetime.datetime.utcnow()
+        offer = factories.OfferFactory()
+        stock = factories.EventStockFactory(offer=offer, beginningDatetime=now)
+        factories.EventStockFactory(offer=offer, beginningDatetime=now + datetime.timedelta(days=1))
```

Comment:
(simplification) tu n'as besoin d'aucun `@freeze_time` dans ton fichier de test parce que le code appelé n'utilise jamais l'effet de bord `datetime.utcnow()`

```suggestion
    def test_filtered_stock_by_date(self):
        # Given
        offer = factories.OfferFactory()
        stock = factories.EventStockFactory(offer=offer)
        factories.EventStockFactory(offer=offer, beginningDatetime=stock.beginningDatetime + datetime.timedelta(days=1))
```